iq/categories/tests.py

```diff
@@ -1,3 +1,137 @@
+import datetime
+from unittest import TestCase, mock
+from unittest.mock import patch
 from django.test import TestCase
+from django.core.urlresolvers import reverse
+from django.utils import timezone
+from django.contrib.auth.models import User
+from model_mommy.mommy import make, prepare
+from iq.questions.models import Question, CategoryQuestion
+from iq.categories.models import Category
+from iq.categories.forms import CategoryForm
+from iq.utils.tests import IqCustomTest
 
-# Create your tests here.
+
+class CategoryModelsTest(IqCustomTest):
+    """
+    Tests category models for appropriate data
+    """
+    def setUp(self):
+        super(CategoryModelsTest, self).setUp()
+
+    def test_category_model(self):
+        c = Category (
+            name = "testing category model name",
+            description = "testing category model description"
+        )
+        c.save()
+        self.assertTrue(c)
+
+
+class CategoryFormsTest(IqCustomTest):
+    """
+    Tests and ensure form validation and invalidation
+    functions properly
+    """
+    def setUp(self):
+        super(CategoryFormsTest, self).setUp()
+        self.client.login(username=self.user.username, password="foo")
+
+    def test_valid_category_form(self):
```

Comment:
The only code you wrote was in the constructor, so that's what you'd want to test (i.e. make sure the form's `instance.created_by` attribute is set to the `created_by` parameter passed into the constructor, etc).